obspy/io/sac/header.py

```diff
@@ -22,7 +22,7 @@
                   when read into sac. [required]''',
        'b': 'F    Beginning value of the independent variable. [required]',
        'e': 'F    Ending value of the independent variable. [required]',
-       'iftype': '''I    Type of file [required]:
+       'iftype': '''I    Type of file [required]:\n
```

Comment:
Hmmm, you're right.  Could've sworn it mattered.  Fixed.  BTW, man those docs take for *ever* to build.